hydradx/model/amm/omnipool_amm.py

```diff
@@ -880,7 +880,7 @@ def calculate_remove_liquidity(state: OmnipoolState, agent: Agent, quantity: flo
     else:
         delta_qa = 0
 
-    if state.withdrawal_fee:
+    if hasattr(state, 'withdrawal_fee') and state.withdrawal_fee > 0:
```

Comment:
This change doesn't seem like it belongs with this PR, why was it made?

Reply:
It's because OmnipoolArchiveState doesn't have a withdraw_fee attribute, so when cash_out_omnipool calls calculate_remove_liquidity, it fails. With this change, it succeeds, and you can use cash_out_omnipool on OmnipoolArchiveState.